src/test/java/codeu/model/store/persistence/PersistentStorageAgentTest.java

```diff
@@ -109,4 +111,13 @@ public void testWriteThroughActivity() {
     persistentStorageAgent.writeThrough(activity);
     Mockito.verify(mockPersistentDataStore).writeThrough(activity);
   }
+
+  @Test
+  public void testDeleteFromMessage() {
+    Message message =
+        new Message(
+            UUID.randomUUID(), UUID.randomUUID(), UUID.randomUUID(), "test content", Instant.now());
+    persistentStorageAgent.deleteFrom(message);
+    Mockito.verify(mockPersistentDataStore).deleteFrom(message);
+  }
```

Comment:
What exactly is this testing?

Reply:
I was a bit confused too, but I tried to follow the existing format. I believe that this code verifies that by calling `persistentStorageAgent.deleteFrom(message);` the datastore's delete function is also called. Basically, it ensures there is a clear connection between the agent and the datastore. I think.

Reply:
Got it.